yarn.lock

```diff
@@ -2137,6 +2182,36 @@
     detect-libc "^1.0.3"
     globby "^11.0.3"
 
+"@parcel/types@2.0.0-beta.3.1":
+  version "2.0.0-beta.3.1"
+  resolved "https://registry.yarnpkg.com/@parcel/types/-/types-2.0.0-beta.3.1.tgz#0bc5255a4454c198d58403b84a66f059106e1610"
+  integrity sha512-8Cmh5/43LZ7wQeqawFMenZ/CCh+YAbyRuwBBApMCDLU3+RSpajUYyTpqa1HZVV8FggQb8E0xvPkmUbj/UHz/gw==
+
+"@parcel/utils@2.0.0-beta.3.1":
+  version "2.0.0-beta.3.1"
```

Comment:
Not sure why Yarn is showing these as I bumped the versions to `rc.0` in in https://github.com/parcel-bundler/parcel/pull/6624/commits/8dfd3d7754dc0ef4a81c982d0b89f44f24074b02?

Reply:
```console
$ yarn list --pattern @parcel/utils
yarn list v1.22.10
├─ @parcel/transformer-js@2.0.0-rc.0
│  └─ self-published@2.0.0-beta.3.1
│     └─ @parcel/utils@2.0.0-beta.3.1
└─ @parcel/utils@2.0.0-rc.0
✨  Done in 1.00s.
```

Reply:
Unrelated changes; happens on my `v2` branch locally too.